websites/D/Discord Turkiye/dist/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "can",
+		"id": "613700645173592086"
+	},
+	"contributors": [
+		{
+			"name": "nowongame",
+			"id": "686990422198845452"
+		}
+	],
```

Comment:
Why did you set yourself as a contributor and not the author? What did the other user do?

Reply:
If they made the code they should be the ones making this PR. If not then you should be the author. Regardless, I see no reason to do this